<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lottery</class>
 <widget class="QMainWindow" name="Lottery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>401</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>情书翩翩体</family>
      <pointsize>48</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>380</y>
      <width>140</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>开始抽取</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/right3.png</normaloff>
      <disabledoff>:/res/right3.png</disabledoff>
      <disabledon>:/res/right3.png</disabledon>
      <activeoff>:/res/right3.png</activeoff>
      <activeon>:/res/right3.png</activeon>:/res/right3.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>480</y>
      <width>140</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/back.png</normaloff>:/res/back.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="Line" name="horizontalLine">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="nameShow">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉呈波波行书</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_renew">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>380</y>
      <width>140</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/return.png</normaloff>:/res/return.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="goTimer">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>480</y>
      <width>140</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>计时界面</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/timer.png</normaloff>:/res/timer.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>2050</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
